frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx

```diff
@@ -76,10 +76,14 @@ export const EncryptionFormGroup: React.FC<EncryptionFormGroupProps> = ({
   }, [encryption.clusterWide, encryption.storageClass, encryptionChecked]);
 
   const toggleEncryption = (checked: boolean) => {
-    setEncryptionDispatch(ActionType.SET_ENCRYPTION, mode, dispatch, {
+    const payload = {
       ...encryption,
       clusterWide: checked,
-    });
+    }
```

Comment:
```suggestion
    };
```

Reply:
You are getting formatting errors hence tests are failing:
```
 /go/src/github.com/openshift/console/frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx
  82:6  error  Insert `;`                                prettier/prettier
  83:7  error  Replace `(!checked)` with `·(!checked)·`  prettier/prettier 
  ```

Reply:
Let me check this